dataactvalidator/config/sqlrules/c23_award_financial_3.sql

```diff
@@ -12,20 +18,19 @@ SELECT
 		FROM award_financial_assistance as sub_afa
 		WHERE submission_id = {0} AND sub_afa.fain = af.fain and
 		COALESCE(sub_afa.assistance_type,'') in ('07','08')) AS original_loan_subsidy_cost_sum
-FROM award_financial AS af
+FROM award_financial_c23_3_{0} AS af
 JOIN award_financial_assistance AS afa
 		ON af.fain = afa.fain
 	  AND af.submission_id = afa.submission_id
-WHERE af.submission_id = {0}
 GROUP BY af.fain
 HAVING
 		(SELECT COALESCE(SUM(sub_af.transaction_obligated_amou::numeric),0) AS transaction_sum
-		FROM award_financial as sub_af WHERE submission_id = {0} AND sub_af.fain = af.fain) <>
+		FROM award_financial as sub_af WHERE sub_af.fain = af.fain) <>
 		(-1*(SELECT COALESCE(SUM(sub_afa.federal_action_obligation),0) AS obligation_sum
 		FROM award_financial_assistance as sub_afa
-		WHERE submission_id = {0} AND sub_afa.fain = af.fain and COALESCE(sub_afa.assistance_type,'') not in ('07','08')) -
+		WHERE sub_afa.fain = af.fain and COALESCE(sub_afa.assistance_type,'') not in ('07','08')) -
 		(SELECT COALESCE(SUM(sub_afa.original_loan_subsidy_cost::numeric),0) AS obligation_sum
 		FROM award_financial_assistance as sub_afa
-		WHERE submission_id = {0} AND sub_afa.fain = af.fain and COALESCE(sub_afa.assistance_type,'') in ('07','08')))
```

Comment:
Still need submission_id here since it's pulling from `award_financial_assistance` not `award_financial`